packages/core/src/formula/formula.ts

```diff
@@ -142,7 +142,23 @@ export type ToddleEnv =
     }
 
 export function isFormula(f: any): f is Formula {
```

Comment:
This might be slightly more accurate
```suggestion
export function isFormula(f: unknown): f is Formula {
```

Reply:
Typescript does not like unknown. It refuses the type property.